FlowCryptAppTests/Functionality/Services/PassPhraseStorageTests/InMemoryPassPhraseStorageTest.swift

```diff
@@ -50,26 +49,28 @@ class InMemoryPassPhraseStorageTest: XCTestCase {
         XCTAssertTrue(passPhraseProvider.passPhrases.isEmpty)
     }
 
-    func testGetPassPhrases() throws {
-        var passPhrases = try sut.getPassPhrases(for: testPassPhraseAccount)
+    func testGetPassPhrases() async throws {
+        var passPhrases = try await sut.getPassPhrases(for: testPassPhraseAccount)
         XCTAssertTrue(passPhrases.isEmpty)
 
         sut.save(passPhrase: testPassPhrase)
 
-        passPhrases = try sut.getPassPhrases(for: testPassPhraseAccount)
+        passPhrases = try await sut.getPassPhrases(for: testPassPhraseAccount)
         XCTAssertTrue(passPhrases.count == 1)
         XCTAssertTrue(passPhrases.contains(
             where: { $0.primaryFingerprintOfAssociatedKey == "11" })
         )
         XCTAssertTrue(passPhrases.filter { $0.date == nil }.isEmpty)
     }
 
-    func testExpiredPassPhrases() {
-        XCTAssertTrue(try sut.getPassPhrases(for: testPassPhraseAccount).isEmpty)
-
-        sut.save(passPhrase: testPassPhrase)
-        sleep(3)
-        XCTAssertTrue(try sut.getPassPhrases(for: testPassPhraseAccount).isEmpty)
+    func testExpiredPassPhrases() async throws {
+//        let passphrase = try await sut.getPassPhrases(for: testPassPhraseAccount)
+//        XCTAssertTrue(passphrase.isEmpty)
+//
+//        sut.save(passPhrase: testPassPhrase)
```

Comment:
@sosnovsky, @tomholub 
I currently have no idea how to implement `testExpiredPassPhrases` test with unit test.
I'm gonna implement it with appium ui test.
Could you tell me if you have better idea using unit test?

Before that, please check PR is implemented in correct way.

Reply:
It's complex behavior involving client configuration, app state etc, I think it would be quite appropriate to test with a ui test.